### DISCUSSION TITLE
Support for DOS Prevention / Banning malicious IP Address?

### DISCUSSION THREAD

**Discussion by spit4520** [wind-c/comqtt#10] [Q&A]
Hi!

I was wondering if there is a mechanism to halt an attack, ie: stop accepting traffic and fire off an event to inform my k8s backplane to it can handle this?

If not, do you have any suggested methods of dealing with this? Getting flooded with CONNAC requests for example?

Thanks!

**Comment by wind-c** [comment]
That's a good question. At present, only simple security protection is implemented. The number of infight Messages is limited by configuration parameters(conf.yml). If the number exceeds the limit, the connection is disconnected or overwritten.
